### PR TITLE
Django 3.2 upgrade

### PR DESCRIPTION
**Pull request by dodget**
This PR currently represents the first step in upgrading this project to Django 3.2 by:

- Upgrading several packaged
- Updating/Adding a few settings
- Changing the AppConfig names for `media_auth` and `media_service`
- Updating a few other minor syntax changes

## Notes

- All tests pass locally

### CODE REVIEW

**Review by Tanvez — APPROVED** [review]
LGTM!

File: media_management_api/settings/base.py [hunk 1]
```diff
@@ -256,7 +256,7 @@
 AWS_S3_KEY_PREFIX = SECURE_SETTINGS["aws_s3_key_prefix"]
 
 # CORS headers
-CORS_ORIGIN_ALLOW_ALL = True
+CORS_ALLOW_ALL_ORIGINS = True
```
Comment (arthurian) [inline media_management_api/settings/base.py:259]: Good catch. Looks like this changed in [django-cors-headers 3.5.0](https://github.com/adamchainz/django-cors-headers/blob/main/HISTORY.rst#350-2020-08-25).

File: media_management_api/media_service/models.py [hunk 2]
```diff
@@ -1,7 +1,6 @@
 from django.db import models, transaction, Error
 from django.db.models import Max
 from django.contrib.auth.models import User
-from django.contrib.postgres.fields import JSONField
```
Comment (arthurian) [inline media_management_api/media_service/models.py:4]: Nice. This was [deprecated in 3.1](https://docs.djangoproject.com/en/3.2/releases/3.1/#postgresql-jsonfield).

**Review by arthurian — APPROVED** [review]
Looks good. Thanks @dodget! :+1:

**Pull request merged**